public/libs/angular-sanitize/angular-sanitize.min.js

```diff
@@ -0,0 +1,16 @@
+/*
+ AngularJS v1.4.7
+ (c) 2010-2015 Google, Inc. http://angularjs.org
+ License: MIT
+*/
+(function(n,h,p){'use strict';function E(a){var f=[];r(f,h.noop).chars(a);return f.join("")}function g(a,f){var d={},c=a.split(","),b;for(b=0;b<c.length;b++)d[f?h.lowercase(c[b]):c[b]]=!0;return d}function F(a,f){function d(a,b,d,l){b=h.lowercase(b);if(s[b])for(;e.last()&&t[e.last()];)c("",e.last());u[b]&&e.last()==b&&c("",b);(l=v[b]||!!l)||e.push(b);var m={};d.replace(G,function(b,a,f,c,d){m[a]=q(f||c||d||"")});f.start&&f.start(b,m,l)}function c(b,a){var c=0,d;if(a=h.lowercase(a))for(c=e.length-
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>Missing semicolon.<br>Expected '{' and instead saw 'd'.<br>'d' is already defined.<br>Expected '{' and instead saw 'for'.<br>Expected '{' and instead saw 'c'.<br>Expected an assignment or function call and instead saw an expression.<br>'c' is already defined.<br>Expected a conditional expression and instead saw an assignment.